sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileClient.java

```diff
@@ -313,92 +312,103 @@ public VoidResponse delete() {
      * @throws StorageErrorException If you attempt to upload a range that is larger than 4 MB, the service returns status code 413 (Request Entity Too Large)
      */
     public Response<FileUploadInfo> upload(ByteBuf data, long length) {
-        return fileAsyncClient.upload(Flux.just(data), length).block();
+        return upload(data, length, 0);
     }
 
     /**
      * Uploads a range of bytes to specific of a file in storage file service. Upload operations performs an in-place write on the specified file.
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * <p>Upload the file from 1024 to 2048 bytes with its metadata and properties and without the contentMD5. </p>
+     * <p>Upload data "default" starting from 1024. </p>
      *
-     * {@codesnippet com.azure.storage.file.fileClient.upload#bytebuf-long-int-filerangewritetype}
+     * {@codesnippet com.azure.storage.file.fileClient.upload#bytebuf-long-int}
      *
      * <p>For more information, see the
      * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/put-range">Azure Docs</a>.</p>
      *
      * @param data The data which will upload to the storage file.
+     * @param length Specifies the number of bytes being transmitted in the request body.
      * @param offset Optional starting point of the upload range. It will start from the beginning if it is {@code null}
-     * @param length Specifies the number of bytes being transmitted in the request body. When the FileRangeWriteType is set to clear, the value of this header must be set to zero.
-     * @param type You may specify one of the following options:
-     * <ul>
-     *      <li>Update: Writes the bytes specified by the request body into the specified range.</li>
-     *      <li>Clear: Clears the specified range and releases the space used in storage for that range. To clear a range, set the Content-Length header to zero.</li>
-     * </ul>
      * @return A response that only contains headers and response status code
      * @throws StorageErrorException If you attempt to upload a range that is larger than 4 MB, the service returns status code 413 (Request Entity Too Large)
      */
-    public Response<FileUploadInfo> upload(ByteBuf data, long length, int offset, FileRangeWriteType type) {
-        return fileAsyncClient.upload(Flux.just(data), length, offset, type).block();
+    public Response<FileUploadInfo> upload(ByteBuf data, long length, int offset) {
+        return fileAsyncClient.upload(Flux.just(data), length, offset).block();
     }
 
     /**
-     * Uploads file to storage file service.
+     * Clears a range of bytes to specific of a file in storage file service. Clear operations performs an in-place write on the specified file.
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * <p>Upload the file from the source file path. </p>
+     * <p>Clears the range starting from 0 bytes with the length of 1024. </p>
```

Comment:
Let's just say clears the first 1024 bytes.

Reply:
Done.